Spigot-Server-Patches/0282-Make-player-data-saving-configurable.patch

```diff
@@ -0,0 +1,35 @@
+From e13748094e37266ddd772ec4e71004af32c19704 Mon Sep 17 00:00:00 2001
+From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
+Date: Mon, 26 Mar 2018 18:30:53 +0300
+Subject: [PATCH] Make player data saving configurable
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index ec89ecfca..6dd3f4698 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -282,4 +282,9 @@ public class PaperConfig {
+     private static void authenticationServersDownKickMessage() {
+         authenticationServersDownKickMessage = Strings.emptyToNull(getString("messages.kick.authentication-servers-down", authenticationServersDownKickMessage));
+     }
++
++    public static boolean savePlayerData = true;
++    private static void savePlayerData() {
++        savePlayerData = getBoolean("settings.save-player-data", savePlayerData);
```

Comment:
Sorry didnt think of this before. We should print a warning if saving is disabled here, so its clear in the logs this is set to false. Then I think were good to pull from there

Reply:
"WARN: Player Data Saving is currently disabled. Any changes to your players data, such as inventories, experience points, advancements and the like will not be saved when they log out."